packages/mui-material-next/src/SnackbarContent/SnackbarContent.types.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
+// eslint-disable-next-line no-restricted-imports
 import { InternalStandardProps as StandardProps, PaperProps, Theme } from '@mui/material';
```

Comment:
`PaperProps['role']` below doesn't seem to explicitly defined: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Paper/Paper.d.ts

Could we type it as `role?: React.AriaRole;` instead?

Reply:
why we should define this type explicitly?
I saw some other components, they don't defined this type explicitly.